metrics/metrics.go

```diff
@@ -32,11 +32,11 @@ const (
 	LabelDDL      = "ddl"
 	LabelGCWorker = "gcworker"
 
-	opSucc   = "op_succ"
```

Comment:
Remind: this may also involves some changes in prometheus dashboard~

Reply:
Yes. Will do.